low_level_platform/api/CMakeLists.txt

```diff
@@ -3,7 +3,7 @@ target_include_directories(lf-low-level-platform-api INTERFACE ${CMAKE_CURRENT_L
 add_library(lf::low-level-platform-api ALIAS lf-low-level-platform-api)
 target_link_libraries(lf-low-level-platform-api INTERFACE lf::tag-api)
 
-if(${CMAKE_SYSTEM_NAME} STREQUAL "Nrf52")
+if(${CMAKE_SYSTEM_NAME} STREQUAL "nRF52")
```

Comment:
We've been using CamelCase for platforms, I think. If instead we want to capitalize platforms the way they are commonly referred to, we should also change `MacOS` to `macOS`. There might be more such cases...

Reply:
I'm not sure our arbitrary policy should override what the devices are called.  Also, why isn't the correct camel case NRF52?  The acronym is spelled out when you say it, which is why OS is capitalized in macOS.

Reply:
Good point. I'm fine with the change, honestly...